server/runtime/src/main/java/org/infinispan/server/configuration/security/OAuth2ConfigurationBuilder.java

```diff
@@ -47,12 +47,16 @@ public OAuth2ConfigurationBuilder introspectionUrl(String introspectionUrl) {
       } catch (MalformedURLException e) {
          throw Server.log.invalidUrl();
```

Comment:
offtopic:
it would be nice to have the URL in the error message
```suggestion
         throw Server.log.invalidUrl(introspectionUrl);
```